### PR TITLE
Import sql to local Docker environment

### PR DESCRIPTION
**Pull request by veljkho**
## Description

In this PR, we're going to add SQL with `100k subscribers`, `5k newsletters` to a local test env.

Note: This PR is final PR for the project k6 implementation. There are some small changes not related to the importing SQL to Docker env, but small changes such as:

Additionally, added a new test just to bypass the onboarding wizard for the first time local execution: `onboardingWizard.js`

I also added taking screenshots in tests. Before, after, or in the middle. Now the screenshots will be included in the artifacts on CircleCI or `_screenshots` folder locally.

Refactored all codes to include `await`. This is very normal in Playwright testing of the E2E suite (on Solaris' side).

There's 1min to import data from the `data.sql` file to local env. I think this is fine for now, until we figure out how to better optimize it. The test run took 3min to complete.

I set the thresholds to pass always, but it's not literally always... If something's wrong it will always fail in the PR checks. The values set are very high but not too much.

## Code review notes

Sorry for the messy commit listing, but please focus on the `file changed`.

If you'd like to test it locally, feel free to execute:

`./do test:performance --scenario=pullrequests`

## Linked tickets

[MAILPOET-4950]

[MAILPOET-4950]: https://mailpoet.atlassian.net/browse/MAILPOET-4950?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/RoboFile.php [hunk 1]
```diff
@@ -319,6 +315,41 @@ public function testPerformance($path = null, $opts = ['url' => null, 'pw' => nu
       ->dir($dir)->run();
   }
 
+  public function testPerformanceSetup() {
+    // get data file URL
+    $url = getenv('WP_TEST_PERFORMANCE_DATA_URL');
```
Comment (samnajian) [inline mailpoet/RoboFile.php:320]: Please add `WP_TEST_PERFORMANCE_DATA_URL` to env.sample as well

**Review by samnajian — CHANGES_REQUESTED** [review]
The test runs properly, but I couldn't find screenshots in `_screenshots`, am I missing something? also please note the comment about the `env`

File: mailpoet/RoboFile.php (hunk 1 above)
Comment (veljkho) [reply mailpoet/RoboFile.php:320]: Done. 👍

**Comment by veljkho** [comment]
> The test runs properly, but I couldn't find screenshots in `_screenshots`, am I missing something? also please note the comment about the `env`

This is weird @samnajian ... What command did you run and what's the output in the end? If you used the one from the description, please see `Checks` line and it should be 14✓ 0X...

**Review by samnajian — APPROVED** [review]
Decided to approve since it's passing on the CI and seems like my local issue.

**Pull request merged**